litestar/middleware/rate_limit.py

```diff
@@ -55,8 +55,14 @@ def __init__(self, app: ASGIApp, config: RateLimitConfig) -> None:
         )
         self.check_throttle_handler = cast("Callable[[Request], Awaitable[bool]] | None", config.check_throttle_handler)
         self.config = config
-        self.max_requests: int = config.rate_limit[1]
-        self.unit: DurationUnit = config.rate_limit[0]
```

Comment:
We'll need to keep these attributes to remain backwards compatible. Otherwise this would be a breaking change.